conans/model/editable_cpp_info.py

```diff
@@ -44,22 +56,26 @@ def load(filepath):
         except configparser.Error as e:
             raise ConanException("Error parsing layout file: %s\n%s" % (filepath, str(e)))
         data = OrderedDict()
+        folders = {}
         for section in parser.sections():
             ref, cpp_info_dir = section.rsplit(":", 1) if ':' in section else (None, section)
+            if cpp_info_dir == "folders":
+                folders[ref] = {k: v for k, v in parser.items(section)}
+                continue
```

Comment:
This `folders` section breaks the previous layout of this file. It might be better to keep those folders under control (we can rename the `cpp_info_dirs` or create and additional one):

```ini
...

[<ref>:build]
/path/...
.../path
```

there is no need to:

```ini
[<ref>:folders]
build=path/at
```

Reply:
The problem is that those folders are unique, not a list of folders. Using the section approach it is very easy to fall in that error, you did in your example.

Reply:
We can check there is only one path here